branchpro/apps/_inference.py

```diff
@@ -56,42 +56,69 @@ def __init__(self):
-                        'appear as bars.'
-                    ]),
-                    html.H6([
-                        'Data must be in the following column format: `Time`, '
-                        '`Incidence number`, `Imported Cases` (optional), '
-                        '`R_t` (true value of R, optional)'
-                    ]),
-                    dcc.Upload(
-                        id='upload-data',
-                        children=html.Div([
-                            'Drag and Drop or ',
-                            html.A('Select Files',
-                                   style={'text-decoration': 'underline'}),
-                            ' to upload your Incidence Number data.'
-                        ]),
-                        style=button_style,
-                        multiple=True  # Allow multiple files to be uploaded
-                    ),
-                    html.Div(id='incidence-data-upload'),
-                    html.H4([
-                        'You can upload your own serial interval here.'
-                    ]),
-                    dcc.Upload(
-                        id='upload-interval',
-                        children=html.Div([
-                            'Drag and Drop or ',
-                            html.A('Select Files',
-                                   style={'text-decoration': 'underline'}),
-                            ' to upload your Serial Interval.'
-                        ]),
-                        style=button_style,
-                        multiple=True  # Allow multiple files to be uploaded
+                    dbc.Row(
+                        [
+                            dbc.Col(
+                                children=[
+                                    html.H6([
+                                        'You can upload your own incidence \
+                                            data here. It will appear as bars \
+                                                .']),
+                                    html.Div([
+                                        'Data must be in the following column '
+                                        'format: `Time`, `Incidence number`, '
+                                        '`Imported Cases` (optional), '
+                                        '`R_t` (true value of R, optional).']),
+                                    dcc.Upload(
+                                        id='upload-data',
+                                        children=html.Div(
+                                            [
+                                                'Drag and Drop or ',
+                                                html.A(
+                                                    'Select Files',
+                                                    style={
+                                                        'text-decoration': 'under\
+                                                            line'}),
```

Comment:
```suggestion
                                                        'text-decoration': 'under'
                                                            'line'}),
```
It doesn't like the line break, at least on some browsers

Reply:
Actually this will fail, what about
```
                                                        'text-decoration':
                                                            'underline'}),
```

Reply:
Okay, I didn't know that!😊